### PR TITLE
feat(leavittbook): creates a titanium-two-line-list-item demo

### LINKED ISSUE #282: Create a titanium-card-two-line-list-item demo
Read through props and make sure there are demos to cover everything.
Make sure the demos make sense and work well. Maybe style them a little bit.

### PR DESCRIPTION
**Pull request by carlycue**
Closes #282

### CODE REVIEW

File: packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts [hunk 1]
```diff
@@ -0,0 +1,153 @@
+/* playground-fold */
+import { css, html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { h1, p } from '@leavittsoftware/titanium-styles';
+
+/* playground-fold-end */
+import '@leavittsoftware/titanium-card/lib/titanium-card-two-line-list-item';
+import '@leavittsoftware/titanium-card';
```
Comment (antwonthegreat) [inline packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts:8]: I think we want the imports that are supplementary to the demo folded, so move titanium-card and mwc-icon up into the fold

File: packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            <titanium-card-two-line-list-item disabled>
+              <img src="https://cdn.leavitt.com/icons/icon-reminder.svg" />
+              <h2>Disabled</h2>
+              <h3>Disabled list item</h3>
+              <mwc-icon>navigate_next</mwc-icon>
+            </titanium-card-two-line-list-item>
+          </card-content>
+        </titanium-card>
+      </div>
+      <h1>Available CDN icons</h1>
```
Comment (aarondrabeck) [inline packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts:59]: We should more this to a root level menu item at some point

Comment (antwonthegreat) [inline packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts:59]: Maybe a separate fold for the icons?

File: packages/titanium-card/src/titanium-card-two-line-list-item.ts [hunk 3]
```diff
@@ -9,9 +9,9 @@ import { TitaniumCardListItemElement } from './titanium-card-list-item';
...
  * @fires item-click - Fired when the item is clicked
  *
- * @cssprop {Color} --app-border-color - Border color
- * @cssprop {Color} --app-dark-text-color - Link & Subheader text color
- * @cssprop {Color} --app-text-color - Link, svg & header text/fill color
+ * @cssprop {Color} [--app-border-color=#dadce0] - Border color
+ * @cssprop {Color} [--app-dark-text-color=#202124] - Link & Subheader text color
+ * @cssprop {Color} [--app-text-color=#5f6368] --app-text-color - Link, svg & header text/fill color
  *
  * @slot - Item content (text)
```
Comment (antwonthegreat) [inline packages/titanium-card/src/titanium-card-two-line-list-item.ts:16]:
```suggestion
 * @slot default - Item content (text)
```

Comment (carlycue) [reply packages/titanium-card/src/titanium-card-two-line-list-item.ts]: [66f84f9](https://github.com/LeavittSoftware/titanium-elements/pull/348/commits/66f84f9f78e15f8d49d78848219c1654665d52be)

File: packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts (hunk 2 above)
Comment (carlycue) [reply packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts:59]: https://github.com/LeavittSoftware/titanium-elements/issues/352

Looks like we'll be doing a separate root level menu item for this.

Comment (carlycue) [reply packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts:59]: https://github.com/LeavittSoftware/titanium-elements/issues/352

Sounds good. Created an issue to tackle this once this PR goes in.

**Review by antwonthegreat — APPROVED** [review]

**Pull request merged**